src/renderer/dx9/renderer.h

```diff
@@ -32,6 +32,16 @@ class Renderer : public BaseRenderer {
 
   uint32_t SubmitTexture(TexFmt format, uint8_t* buffer, int width,
                          int height) override;
+  std::vector<uint8_t> GetImageFromTexture( uint32_t texture,
```

Comment:
Some formatting weirdness here and below. Please reformat with clang-format we usually use